api/tests/Feature/TeamsTest.php

```diff
@@ -216,43 +244,120 @@ public function testTeamUpdateMutation(): void
       }
       ',
       [
-        'id' => $teamOne->id,
+        'id' => $this->team1->id,
         'team' => [
-          'name'=> 'team one',
+          'name'=> 'new_team_name',
           'displayName' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Name EN',
+            'fr' => 'New Team Name FR',
           ],
           'description' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Description EN',
+            'fr' => 'New Team Description FR',
           ],
-          'contactEmail' => 'test@test.com',
+          'contactEmail' => 'newContactEmail@test.com',
           'departments' => [
-            "sync" => [$departmentId],
+            'sync' => [$departmentId],
           ],
         ]
       ]
     )->assertJson([
       'data' => [
         'updateTeam' => [
-          'name' => 'team one',
+          'name'=> 'new_team_name',
           'displayName' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Name EN',
+            'fr' => 'New Team Name FR',
           ],
           'description' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Description EN',
+            'fr' => 'New Team Description FR',
           ],
-          'contactEmail' => 'test@test.com',
+          'contactEmail' => 'newContactEmail@test.com',
           'departments' => [
-            [
-              'id' => $departmentId,
-            ],
+            ['id' => $departmentId],
           ],
         ]
       ]
     ]);
   }
+
+  public function testTeamDeleteMutation(): void
+  {
+    // Detach departments before deleting team
+    $this->team3->departments()->detach();
+
+    // Assert team update successful across all input fields
+    $this->actingAs($this->admin, "api")->graphQL(
+      /** @lang GraphQL */
+      '
+      mutation deleteTeam($id: UUID!) {
+          deleteTeam(id: $id) {
+            id
+          }
+      }
+      ',
+      [ 'id' => $this->toBeDeletedUUID, ]
+    )->assertJson([
+      'data' => [
+        'deleteTeam' => [
+          'id' => $this->toBeDeletedUUID,
+        ],
+      ]
+    ]);
+  }
+
+  public function testViewAnyTeamMembers(): void
+  {
+    // Attach all users to team one
+    $this->admin->attachRole("pool_operator", $this->team1);
+    $this->poolOperator1->attachRole("pool_operator", $this->team1);
+    $this->poolOperator2->attachRole("pool_operator", $this->team1);
+
+    // Assert all teams query contains expected results
+    $query = $this->actingAs($this->admin, "api")
```

Comment:
In addition to testing with admin, try acting as poolOperators from the same or different teams.